<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_in_epsilon?' type='fail'>
    <section_ name='Args=[3, 2, 0, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_in_epsilon?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Integer' value='3'/>
        <actual_ class='Integer' value='2'/>
        <epsilon_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected |3 - 2| (1) to be &lt;= 0.'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[4, 3, 0, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_in_epsilon?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Integer' value='4'/>
        <actual_ class='Integer' value='3'/>
        <epsilon_ class='Integer' value='0'/>
        <exception_ class='Minitest::Assertion' message='Expected |4 - 3| (1) to be &lt;= 0.'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
